typing/ctype.ml

```diff
@@ -1134,7 +1134,7 @@ let rec copy ?partial ?keep_names scope ty =
   let copy = copy ?partial ?keep_names scope in
   let ty = repr ty in
   match ty.desc with
-    Tsubst ty -> ty
+    Tsubst (ty, _) -> ty
```

Comment:
here we can assert  `= None` if I understand correctly?

Reply:
Asserting that does not break the test.
However, the first argument of `Tsubst` is always a copy of the current node, so it is safe (or conservative) to return it.